packages/vue/src/components/organisms/SfGroupedProduct/SfGroupedProduct.stories.js

```diff
@@ -385,7 +385,7 @@ storiesOf("Organisms|GroupedProduct", module)
           :price-special="priceSpecial"
       >
         <template #image="{ image, title, imagePlaceholder, imageLazy, imagePictureBreakpoint }">
-          <div :style="{height: '111px'}">CUSTOM IMAGE</div>
+          <div :style="{height: '111px', 'line-height': '1'}">CUSTOM IMAGE</div>
```

Comment:
@justyna-13 is it possible to fix this inside the component? @pspaczek

Reply:
@martaradziszewska @justyna-13 to fix this case we should fix SfGroupedProduct, than we can remove line-height from these stories :)

probably it should be helpfull for you @justyna-13 
``` scss
.sf-grouped-product-item__aside {
line-height: 0 // remove it
display: flex // add it
}
```